client/src/Components/Navbar.jsx

```diff
@@ -0,0 +1,162 @@
+    const savedLang = localStorage.getItem('i18nextLng') || 'en';
+    i18.changeLanguage(savedLang);
+  },[])
+  
+  const handleLanguageChange = (lang) => {
+    const currentPath = window.location.pathname;
+    const newPath = currentPath.replace(`/${i18.language}`, `/${lang}`);
+    i18.changeLanguage(lang);
+    localStorage.setItem('i18nextLng', lang);
+    navigate(newPath);
+    setDropdownVisible(false);
+  };
+  
+  // const getLanguagePath = () => `/${i18.language}`
+
+  if(isLoading)
+  {
+    return <div>Loading...</div>
+  }
+
+  return (
+    <div className='gap-[1150px]'>
+
+      {/* This is the 1st navbar */}
+      <div className= "bg-green-700}">
+        <div className={`flex bg-green-700 gap-[1150px] ${i18.language === 'ur' ? 'gap-[1130px]' : 'gap-[1150px'}
+                                                          ${i18.language === 'de' ? 'gap-[1090px]' : 'gap-[1140px'}`}>
+          <div className="flex flex-row items-center">
+            <Link to={`/${i18.language}`}>
+              <h3 className='text-white mr-3 ml-4'>DeUrdu</h3>
+            </Link>
+            <Link to={`/${i18.language}`}>
+              <h3 className='text-white'> ڈی اردو </h3>
+            </Link>
+          </div>
+            <div className="flex gap-1">
+               {/* Globe button with onClick to toggle dropdown */}
+              <button 
+                className='px-2 py-0.5 m-1 rounded-lg text-white relative'
+                onClick={() => setDropdownVisible(!dropdownVisible)}
+              >
+                <FontAwesomeIcon icon={faGlobe} className='mr-1' />
+                {i18.language.toUpperCase()}
+              </button>
+
+              {/* Dropdown menu */}
+              {dropdownVisible && (
+                <div className='absolute mt-2 bg-gray-200 rounded-md shadow-lg'>
+                  <button 
+                    className='block px-4 py-2 text-black hover:bg-gray-300'
+                    onClick={() => handleLanguageChange('en')}
+                  >
+                    English (EN)
+                  </button>
+                  <button 
+                    className='block px-4 py-2 text-black hover:bg-gray-300'
+                    onClick={() => handleLanguageChange('de')}
+                  >
+                    German (De)
+                  </button>
+                  <button 
+                    className='block px-[25px] py-2 text-black hover:bg-gray-300'
+                    onClick={() => handleLanguageChange('ur')}
+                  >
+                    Urdu (UR)
+                  </button>
+                </div>
+              )}
+              {/* <Link to={`${getLanguagePath()}/login`}>
+                <button className='px-2 py-0.5 m-1 rounded-lg text-white'>{login}</button>
+              </Link>
+              <Link to={`${getLanguagePath()}/register`}>
+                <button className='px-2 py-0.5 m-1 rounded-lg text-white'>{register}</button>
+              </Link> */}
```

Comment:
I see commented out code in many places, not sure if this is debugging leftover or not needed any longer

Reply:
No, the commented code is for no needed longer.
But, in-case in in future, we think of migrating our webapp from auth0 to JWT Tokens.
It would be beneficial.